### DISCUSSION TITLE
请求为Mac的Safari做一款这个插件

### DISCUSSION THREAD

**Discussion by Kanbebetter** [BewlyBewly/BewlyBewly#555] [Ideas]
能不能给Safari也做一下这个插件，我也很愿意去付费购买这个

**Comment by hakadao** [comment]
十分抱歉 之前已经有人safari支持了
https://github.com/hakadao/BewlyBewly/pull/504
这段时间我会编译一个版本写个安装教程

**Reply by Kanbebetter** [reply]
好的谢谢，我现在十分想要在Safari上用这个插件

**Comment by MengNianxiaoyao** [comment]
> 好的谢谢，我现在十分想要在Safari上用这个插件

safari已经构建出了第一个插件包，可以在这里[下载](https://github.com/BewlyBewly/BewlyBewly/actions/runs/8832683924/artifacts/1447091849)

**Comment by hakadao** [comment]
可參考這個 https://github.com/BewlyBewly/BewlyBewly/issues/679
